README.md

```
@@ -77,47 +78,43 @@ Notion: [click](https://www.notion.so/GOAT-27b6550d6d4480019236d1a9d4c9ef54?sour
+├── styles/       # CSS 스타일 파일
+└── assets/       # 정적 리소스 (이미지 등)
+```
+
+## 🧪 테스트
+
+프로젝트는 Vitest와 React Testing Library를 사용하여 테스트합니다.
+
+### 테스트 커버리지
+
+| 분류 | 테스트 파일 |
+|------|------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)